### PR TITLE
Add contributing md 120

### LINKED ISSUE #120: Add a CONTRIBUTING.md to the Ops repo
### Overview

As developers we want to enhance the `ops repo` by adding a CONTRIBUTING.md file. This guide will provide clear instructions for new volunteers on how to contribute to the ops repo. By giving these guidelines, we streamline the contribution process and create a welcoming environment for collaboration.

### Action Items

In the CONTRIBUTING.md guide provide instructions for
- [x] Making contributions to the repo
- [x] Cloning the [ops](https://github.com/hackforla/ops) repo
- [x] Creating a new branch to work on the feature, make sure you provide instructions for the [naming convention](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27b-working-on-an-issue-2-create-a-new-branch-where-you-will-work-on-your-issue)
- [x] Adding, committing and pushing the changes
- [x] Creating a PR

### Resources/Instructions

Example for contributing guide: https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md
Link to the naming conventions: https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27b-working-on-an-issue-2-create-a-new-branch-where-you-will-work-on-your-issue

### PR DESCRIPTION
**Pull request by awlFCCamp**
Fixes #120 
Added a contributing markdown file to provide clear instructions for new contributors. Added instructions for making contributions to the repo: forking repo, cloning repo, creating a new branch, adding committing, pushing changes, and creating a pull request.

### CODE REVIEW

**Comment by awlFCCamp** [comment]
@chelseybeck @nyarly Please take a look at it, and make any suggestions for revision. Thank you

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+
+```bash
+git remote add upstream https://github.com/hackforla/ops.git
+```
+
+<sub>[Back to Table of Contents](#table-of-contents)</sub>
+
+### **Working on an issue**
+
+For each issue you address, create a new branch. Working on topic branches ensures that your main branch(named `master`)remains untouched and makes it easier to synchronize your fork with the main project.
```
Comment (chelseybeck) [inline CONTRIBUTING.md:150]:
```suggestion
For each issue you address, create a new branch. Working on topic branches ensures that your main branch(named `main`)remains untouched and makes it easier to synchronize your fork with the main project.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+
+#### **Working on an issue (1): Verify current branch is `master`**
```
Comment (chelseybeck) [inline CONTRIBUTING.md:152]:
```suggestion
#### **Working on an issue (1): Verify current branch is `main`**
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+
+first make sure you are on the master branch.
```
Comment (chelseybeck) [inline CONTRIBUTING.md:154]:
```suggestion
first make sure you are on the main branch.
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,253 @@
...
+
+```bash
+git checkout master
```
Comment (chelseybeck) [inline CONTRIBUTING.md:157]:
```suggestion
git checkout main
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,253 @@
...
+```
+
+Update your master branch with the latest changes
```
Comment (chelseybeck) [inline CONTRIBUTING.md:160]:
```suggestion
Update your main branch with the latest changes
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,253 @@
...
+
+```
+git pull upstream master
```
Comment (chelseybeck) [inline CONTRIBUTING.md:163]:
```suggestion
git pull upstream main
```

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,253 @@
...
+**IMPORTANT:** Before you push your local commits to your repository, sync your fork with the `master` branch of the `origin` remote repository. Use `git pull upstream` to ensure your local repository is up-to-date with the `master` branch of the `origin` remote repository.
+
+```bash
+git pull upstream
+```
+
+After committing your changes locally, you can use below commands to push your committed changes to a remote repository, making them accessible to others working on the same projects.
+
+```bash
+git push --set-upstream add-contributing-md-120
```
Comment (chelseybeck) [inline CONTRIBUTING.md:238]:
```suggestion
git push --set-upstream origin add-contributing-md-120
```

**Comment by chelseybeck** [comment]
@awlFCCamp I recommended a few changes. GitHub used to default to naming branches 'master', but switched to 'main'. Newer repos (like ours) use 'main'. Other than that, this looks great! Thanks for working on it 😄

**Comment by awlFCCamp** [comment]
@chelseybeck I made changes. Please take a look at it, and make any suggestions for revision. Thank you.

**Review by abbyz123 — APPROVED** [review]
Looks good to me. Thank you so much for your work!

**Pull request merged**